hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -10,14 +10,17 @@ parameters:
   value: staging
 - name: IMAGE_TAG
   value: latest
-- name: EXTERNAL_DNS_AWS_CREDS
-  value: external-dns-route53-admin
-- name: EXTERNAL_DNS_MANAGED_DOMAIN
-  value: s1.openshiftapps.com
```

Comment:
i propose we keep those around to avoid failing on redundant parameters in the saas repo :man_facepalming:

Reply:
These should not be present in the public Hive repo IMO. ACM folks picked them up accidentally and then were asking what they were used for.

Reply:
removed